workflow.src

```diff
@@ -0,0 +1,129 @@
+    </li>
+    <li>
+      <p>
+        Once your authenticated on the repo page, a new button should appear on the page to create a new
+        pull request (<code>Compare &amp; pull request</code>). Go ahead and punch it.
+      </p>
+      <p><img src="img/create-a-pr.png" alt="Compare &amp; pull request" /></p>
+    </li>
+    <li>
+      <p>
+        Make sure you like the title of your pull request, add a proper description (including mentioning the contributors with
+        <a href="https://github.com/w3c/licenses/blob/master/SW-CONTRIBUTING.md">"+@username"</a>), and use
+        <a href="issue-metadata.html">proper GitHub labels</a>, add one or more reviewers so they get pinged.
+        Once you're satisfied, create the pull request. Note that you can update all of that information
+        after that if needed.
+      </p>
+    </li>
+    <li>
+     <p>
+       ALL normative spec changes are generally expected to have a corresponding pull request in
+       <a href="https://github.com/w3c/web-platform-tests/">web-platforms-tests</a>,
+       either in the form of new tests or modifications to existing tests, or must include the rationale for why test
+       updates are not required for the proposed update.
+     </p>
+     <p>
+       Typically, both pull requests (spec updates and tests) will be merged at the same time. If a pull request
+       for the specification is approved but the other needs more work, add the
+       '<a href="https://w3c.github.io/issue-metadata.html">needs tests</a>' label or,
+       in web-platform-tests, the
+       '<a href="https://github.com/w3c/web-platform-tests/issues?utf8=%E2%9C%93&q=label%3Astatus%3Aneeds-spec-decision%20">status:needs-spec-decision</a>'
+       label. Note that a test change that contradicts
+       the specification should not be merged before the corresponding specification change.
+     </p>
+     <p>
+       If testing is not practical due to <a href="https://github.com/w3c/web-platform-tests/">web-platforms-tests</a> limitations,
+       please explain why and if appropriate file an issue with the
+       '<a href="https://github.com/w3c/web-platform-tests/issues?utf8=%E2%9C%93&q=label%3Atype%3Auntestable%20">type:untestable</a>'
+       label to follow up later.
+     </p>
+    </li>
+    <li>
+      <p>
+        Wait for the needed reviews. All pull requests must have been reviewed by one or more participants of the working group.
+        They can use approve changes, give you <code>+1</code>, comment, etc. You made need to make additional commits in your branch and push them
+        again in order to address the review. The pull request will keep tracking the branch so it will get automatically updated with the new commits.
+      </p>
+    </li>
+    <li>
+      <p>
+        Once the pull request gets enough support, it can be merged (use the <code>Squash and merge</code> to avoid cluttering the
```

Comment:
For squash and merge, can you also suggest cleaning up the commit message? The one that GitHub generates will usually be terrible.